examples/hybrid_jobs/7_Running_notebooks_as_hybrid_jobs/src/notebook_runner.py

```diff
@@ -79,7 +79,7 @@ def convert_jobs_hyperparams_to_pm_params(braket_hyperparams: dict[str, str]) ->
     return papermill_params
 
 
-def get_notebook_name(input_dir: PathLike) -> str:
+def get_notebook_name[T: PathLike](input_dir: T) -> str:
```

Comment:
As much as I _love_ these 3.12 features, notebooks should have Python 3.11-compatible syntax (unless these are job _decorator_ notebooks, in which case you can disregard this comment.